docs/versioning.md

```
@@ -0,0 +1,40 @@
+---
+title: "Versioning"
+---
+
+Home Assistant Core uses [calendar versioning](https://calver.org/) (CalVer). A version is made up of three dot-separated parts:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)